src/Application/Messenger/Handler/StripePayoutHandler.php

```diff
@@ -22,6 +22,10 @@
  */
 class StripePayoutHandler implements MessageHandlerInterface
 {
+    private const MAX_RETRY_DEPTH = 10;
+    /** @var string[] */
+    private static array $processedPayoutIds = [];
```

Comment:
Why is this static? Isn't there only ever going to be up to one instance of this class per process anyway (outside tests), in which case an instance property would do the same job. We never instantiate StripePayoutHandler ourselves, we let the container do it which afaik will not make more than one.

Reply:
True, no good reason. Changed.